src/front-door/azext_front_door/custom.py

```diff
@@ -423,25 +430,44 @@ def remove_fd_backend(cmd, resource_group_name, front_door_name, backend_pool_na
     client.create_or_update(resource_group_name, front_door_name, frontdoor).result()
 
 
-def create_fd_health_probe_settings(cmd, resource_group_name, front_door_name, item_name, path, interval,
-                                    protocol=None):
+def create_fd_health_probe_settings(cmd, resource_group_name, front_door_name, item_name, probe_path, probe_interval,
```

Comment:
Is this command usable before this PR?

Reply:
Yes. interval and path were options_list in _params.py
c.argument('probe_protocol', options_list='--protocol', arg_type=get_enum_type(FrontDoorProtocol), help='Protocol to use for sending probes.')
        c.argument('probe_interval', options_list='--interval', type=int, help='Interval in seconds between probes.')